src/Compute/Compute.Autorest/docs/Invoke-AzSpotPlacementScore.md

```diff
@@ -14,32 +14,45 @@ Generates placement scores for Spot VM skus.
 
 ### PostExpanded (Default)
 ```
-Invoke-AzSpotPlacementRecommender -Location <String> [-SubscriptionId <String>] [-AvailabilityZone]
+Invoke-AzSpotPlacementScore -Location <String> [-SubscriptionId <String>] [-AvailabilityZone]
  [-DesiredCount <Int32>] [-DesiredLocation <String[]>] [-DesiredSize <IResourceSize[]>]
  [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
 ```
 
 ### Post
 ```
-Invoke-AzSpotPlacementRecommender -Location <String>
- -SpotPlacementRecommenderInput <ISpotPlacementRecommenderInput> [-SubscriptionId <String>]
- [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+Invoke-AzSpotPlacementScore -Location <String> -SpotPlacementScoresInput <ISpotPlacementScoresInput>
+ [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### PostViaIdentity
+```
+Invoke-AzSpotPlacementScore -InputObject <IComputeIdentity>
+ -SpotPlacementScoresInput <ISpotPlacementScoresInput> [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### PostViaIdentityExpanded
+```
+Invoke-AzSpotPlacementScore -InputObject <IComputeIdentity> [-AvailabilityZone] [-DesiredCount <Int32>]
+ [-DesiredLocation <String[]>] [-DesiredSize <IResourceSize[]>] [-DefaultProfile <PSObject>] [-Confirm]
+ [-WhatIf] [<CommonParameters>]
```

Comment:
If this cmdlet is in preview, please explain that in the Description section. If there is another unified public doc about this feature, you can put a link here too.

Reply:
Updated description to include that this cmdlet is in preview and linked the public doc